### PR TITLE
DATAACCESS: Write AdminRepository Create method

### PR DESCRIPTION
**Pull request by javohirtohirovich**

### CODE REVIEW

File: src/Eclo.DataAccess/Repositories/Admins/AdminRepository.cs [hunk 1]
```diff
@@ -28,9 +28,31 @@ public async Task<long> CountAsync()
...
+        try
+        {
+            await _connection.OpenAsync();
+            
+            string query = "INSERT INTO public.admins(first_name, last_name, phone_number, phone_number_confirmed, " +
+                 "password_hash, salt, image_path, passport_serial_number, " +
+                     "birth_date, region, district, address, created_at, updated_at) " +
+                 "VALUES (@FirstName, @LastName, @PhoneNumber, @PhoneNumberConfirmed, " +
+                     "@PasswordHash, @Salt, @ImagePath, @PassportSerialNumber, @BirthDate, " +
+                         "@Region, @District, @Address, @CreatedAt, @UpdatedAt);";
```
Comment (ShohruhUzDev) [inline src/Eclo.DataAccess/Repositories/Admins/AdminRepository.cs:42]: har bir yangi qator tashlanganda tab tashlanishi kerak

**Review by ShohruhUzDev — CHANGES_REQUESTED** [review]

**Review by Maxsetovich — APPROVED** [review]
Completed AdminRepository

**Review by Maxsetovich — APPROVED** [review]
fixes are resolved

**Review by Maxsetovich — APPROVED** [review]
fixes are resolved

**Pull request merged**